inc/saiqueue.h

```diff
@@ -81,111 +79,108 @@ typedef enum _sai_queue_attr_t
     /* -- */
 
     SAI_QUEUE_ATTR_END,
-    
+
     /* Custom range base value */
     SAI_QUEUE_ATTR_CUSTOM_RANGE_START = 0x10000000,
 
     /* --*/
     SAI_QUEUE_ATTR_CUSTOM_RANGE_END
 
-
 } sai_queue_attr_t;
 
-
-
 /**
  * @brief Enum defining statistics for Queue.
  */
 
 typedef enum _sai_queue_stat_counter_t
```

Comment:
Any change is fine by me as long as we keep it consistent. Please take time to review this change and let me know what else should we keep/change